Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/SessionWrapper.java

```diff
@@ -0,0 +1,36 @@
+package nl.soccar.gameserver.model.session;
+
+import nl.soccar.library.Session;
+
+/**
+ * @author Lesley
+ */
+public final class SessionWrapper {
+
+    private final Session session;
+    private final RoomWrapper room;
+    private final GameWrapper game;
+
+    public SessionWrapper(Session session) {
+        this.session = session;
+
+        room = new RoomWrapper(this, session.getRoom());
+        game = new GameWrapper(this, session.getGame());
+    }
+
+    public void destroy() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)